app/src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string-array name="alarm_time_titles">
+        <item>at start time</item>
+        <item>5 minutes</item>
+        <item>10 minutes</item>
+        <item>15 minutes</item>
+        <item>30 minutes</item>
+        <item>45 minutes</item>
+        <item>60 minutes</item>
+    </string-array>
+
+    <string-array name="alarm_time_values" translatable="false">
+        <item>0</item>
+        <item>5</item>
+        <item>@string/default_alarm_time_value</item>
```

Comment:
This looks confusing to me. What's the purpose of replacing one of the strings with the arbitrary default alarm time?